docs/features/help.rst

```diff
@@ -1,11 +1,58 @@
 Help
 ====
 
-use the categorize() function to create help categories
+From our experience, end users rarely read documentation no matter how high-
+quality or useful that documentation might be.  So it is important that you
+provide good built-in help within your application.  Fortunately, ``cmd2``
+makes this easy.
 
-Use ``help_method()`` to custom roll your own help messages.
+Getting Help
+------------
 
-See :ref:`features/argument_processing:Help Messages`
+``cmd2`` makes it easy for end users of ``cmd2`` applications to get help via
+the built-in ``help`` command.  The ``help`` command by itself displays a list
+of the commands available:
+
+.. code-block:: text
+
+    (Cmd) help
+
+    Documented commands (use 'help -v' for verbose/'help <topic>' for details):
+    ===========================================================================
+    alias  help     ipy    py    run_pyscript  set    shortcuts
+    edit   history  macro  quit  run_script    shell
+
+The ``help`` command can also be used to provide detailed help for a specific
+command:
+
+.. code-block:: text
+
+    (Cmd) help quit
+    Usage: quit [-h]
+
+    Exit this application
+
+    optional arguments:
+      -h, --help  show this help message and exit
+
+Providing Help
+--------------
+
+``cmd2`` makes it easy for developers of ``cmd2`` applications to provide this
+help.  By default, the help for a command is the docstring for the ``do_*``
+method defining the command - e.g. for a command **foo**, that command is
+implementd by defining the ``do_foo`` method and the docstring for that method
+is the help.
+
+For commands which use one of the ``argparse`` decorators to parse arguments,
+help is provided by ``argparse``. See
+:ref:`features/argument_processing:Help Messages` for more information.
+
+Occasionally there might be an unusual circumstance where providing static help
```

Comment:
I don't think this applies to argparse commands

Reply:
Interesting. Honestly I'm not sure.  I'll double-check and clarify it one way or the other.